studiopress/pattern-manager
# Block theme updates if they have been modified by PM

This PR makes it so that if a theme has been modified by Pattern Manager, it will not be available for auto-update within the WordPress dashboard. This is to prevent people accidentally wiping out their patterns. 

### How to test
1. Get your WordPress into a state where a theme needs an update, and check the "Dashboard" > "Updates" page to see that an update is available. Do not update the theme though.
2. Now, add a pattern to the theme, or delete one from the theme. 
3. Re-check your updates page. Notice the update is no longer available.

## Before adding a pattern with PM
<img width="992" alt="Screen Shot 2023-03-10 at 2 31 23 PM" src="https://user-images.githubusercontent.com/7538525/224415820-a43d6a79-61a2-437a-92c0-bdd48b23a63e.png">

## After adding a pattern with PM
<img width="1790" alt="Screen Shot 2023-03-10 at 2 33 07 PM" src="https://user-images.githubusercontent.com/7538525/224416092-5df4eadd-8c3a-462c-a197-6f014f894025.png">


### Documentation




### Suggested changelog entry

- 

### Notes
